flink-runtime/src/main/java/org/apache/flink/runtime/operators/sort/SpillingThread.java

```diff
@@ -586,4 +594,8 @@ private void getSegmentsForReaders(
 			}
 		}
 	}
+
+	private IOException wrapWithIOException(InterruptedException e) throws IOException {
+		return new IOException("The sortes has been interrupted", e);
```

Comment:
Typo `Sortes` -> `Sorter` ?

I'm a little wondering that is it also possible to direct throws `InterruptedException` (We might add it to some methods' declaration) ? 

If there are blockers for throwing `InterruptedException` directly I think we might promote the method to the base class since the other threads also faced with the situation to wrapper.